### PR TITLE
Alter column with default value

### PR DESCRIPTION
**Pull request by wyxxxcat**
supplement： https://github.com/selectdb/ccr-syncer/pull/362

### CODE REVIEW

File: pkg/ccr/base/spec.go [hunk 1]
```diff
@@ -1254,8 +1254,7 @@ func (s *Spec) LightningSchemaChange(srcDatabase, tableAlias string, lightningSc
 	sql = strings.Replace(sql, "REPLACE_IF_NOT_NULL NULL DEFAULT \"null\"",
 		"REPLACE_IF_NOT_NULL NULL DEFAULT NULL", 1)
 
-	sql = strings.ReplaceAll(sql, "DEFAULT \"CURRENT_TIMESTAMP\"", "DEFAULT CURRENT_TIMESTAMP")
-	sql = strings.ReplaceAll(sql, "DEFAULT \"BITMAP_EMPTY_DEFAULT_VALUE\"", "DEFAULT BITMAP_EMPTY_DEFAULT_VALUE")
+	sql = HandleDefaultValue(lightningSchemaChange)
```
Comment (w41ter) [inline pkg/ccr/base/spec.go]:
```suggestion
	sql = HandleSchemaChangeDefaultValue(lightningSchemaChange)
```

File: pkg/ccr/base/spec.go [hunk 2]
```diff
@@ -1614,3 +1613,19 @@ func FilterUnsupportedProperties(modifyProperty *record.ModifyTableProperty) map
 	}
 	return validProperties
 }
+
+func HandleDefaultValue(lightningSchemaChange *record.ModifyTableAddOrDropColumns) string {
```
Comment (w41ter) [inline pkg/ccr/base/spec.go]:
```suggestion
func HandleSchemaChangeDefaultValue(lightningSchemaChange *record.ModifyTableAddOrDropColumns) string {
```

**Review by w41ter — APPROVED** [review]

File: pkg/ccr/base/spec_test.go [hunk 3]
```diff
@@ -126,3 +126,71 @@ func TestCheckModifyTablePropertySql(t *testing.T) {
...
+						{Name: "test", Type: record.ColumnType{Type: "VARCHAR", Len: 10}, DefaultValue: "CURRENT_TIMESTAMP"},
+					},
+				},
+			},
+			expect: "ALTER TABLE `t` ADD COLUMN `test` VARCHAR(10) NOT NULL DEFAULT \"CURRENT_TIMESTAMP\" COMMENT 'test'",
+		},
+	}
+
+	for i, c := range testCases {
+		if actual := base.HandleDefaultValue(&c.origin); actual != c.expect {
```
Comment (w41ter) [inline pkg/ccr/base/spec_test.go]:
```suggestion
		if actual := base.HandleSchemaChangeDefaultValue(&c.origin); actual != c.expect {
```
